### PR TITLE
updating alternate path for java in u16 base image  

### PR DESCRIPTION
**Pull request by Ranjansingh41**
https://github.com/Shippable/pm/issues/11421

### CODE REVIEW

File: java/install.sh [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-sudo update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac 1
-sudo update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
-sudo update-alternatives --set javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
-
-
 echo "================ Installing oracle-java11-installer ================="
 export ORACLEJDK_VERSION=11
 mkdir -p /usr/lib/jvm && cd /usr/lib/jvm
-wget --no-check-certificate -c --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/11+28/55eed80b163941c8885ad9298e6d786a/jdk-"$ORACLEJDK_VERSION"_linux-x64_bin.tar.gz
+wget --no-check-certificate -c --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/11+28/55eed80b163941c8885ad9298e6d786a/jdk-11_linux-x64_bin.tar.gz
```
Comment (rageshkrishna) [inline java/install.sh]: Should this also come from the env var?

File: java/install.sh [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+echo 'export PATH=$PATH:/usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin' >> /etc/drydock/.env
+
+sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin/java 1
+sudo update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin/javac 1
+
+
+echo "================ Installing openjdk11-installer ================="
+sudo add-apt-repository ppa:openjdk-r/ppa
+sudo apt-get update
+sudo apt install -y openjdk-11-jdk --allow-unauthenticated
```
Comment (rageshkrishna) [inline java/install.sh]: Version should be in an env var.

Comment (Ranjansingh41) [reply java/install.sh]: ok

File: java/install.sh [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
-echo 'export JAVA_HOME=/usr/lib/jvm/java-11-oracle' >> /etc/drydock/.env
-echo 'export PATH=$PATH:/usr/lib/jvm/java-11-oracle/jre/bin' >> /etc/drydock/.env
+mv jdk-"$ORACLEJDK_VERSION"/ java-"$ORACLEJDK_VERSION"-oraclejdk-amd64
+echo 'export JAVA_HOME=/usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64' >> /etc/drydock/.env
+echo 'export PATH=$PATH:/usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin' >> /etc/drydock/.env
+
+sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin/java 1
+sudo update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-"$ORACLEJDK_VERSION"-oraclejdk-amd64/bin/javac 1
+
+export OPEN_JDK=11
```
Comment (rageshkrishna) [inline java/install.sh]: This should be called `OPENJDK_VERSION` so it's consistent with `ORACLEJDK_VERSION`.

**Pull request merged**